ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/orchestration.md

```diff
@@ -68,6 +68,7 @@ This doc is to show how to setup OCI Events and Functions to make them work in s
 
 ### Prepare an object storage bucket.
 After you created the bucket, don't foregt to enable `Emit Object Events` under bucket details page. Without this, Events Service won't be able to catch any object related events. 
+
```

Comment:
It is necessary because this illustration is one the same line with text, which I feel looks messy:
https://github.com/bug-catcher/oci-data-science-ai-samples/blob/master/ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/orchestration.md